### ISSUE TITLE
Implement slide layouts functionality

### ISSUE THREAD

**Issue by vanjuxin-dju** [vanjuxin-dju/filmstrip.js#25]
The idea is to implement functionality that allows you to implement such slide layouts as:

- a regular slide (with a title and a full width content or a subtitle)
- a comparison slide (with two columns of content)
- a picture slide (an image fills full slide view, the slide can contain description)

**Comment by vanjuxin-dju** [comment]
Implemented picture slide functionality.
You can make a picture slide with the following markup:
```
<section class="slide">
    <div class="slide-content">
        <p class="p-image full-slide"><img src="an_image_url"></p><!-- full-slide class -->
        <p class="p-description">This is a description for the image you added.</p> <!-- this p tag can also have p-small or p-big class -->
    </div>
</section>
```

**Comment by vanjuxin-dju** [comment]
Fixed applying of styles: if `full-slide` class isn't added to the `p.p-image` element, `p-description` styles won't be applied.

**Comment by vanjuxin-dju** [comment]
Implemented two-column slide functionality.
You can make a comparison slide with the following markup:
```
<section class="slide">
    <div class="slide-content">
        <h1 class="text-center">Pros and cons</h1>
        <ul class="plus-list two-column column-left"><!-- two-column and column-left classes -->
            <li>one</li>
            <li>two</li>
        </ul>
        <ul class="minus-list two-column column-right"> <!-- two-column and column-right classes -->
            <li>three</li>
            <li>four</li>
            <li>five</li>
        </ul> <!-- if you add some content without two-column class and column-left or column-right class, it will be rendered as a full-width content -->
    </div>
</section>
```

**Comment by vanjuxin-dju** [comment]
Implemented in pull request #26
